SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Reporting a security issue
+
+Please email us directly at [otw-coders@transformativeworks.org](mailto:otw-coders@transformativeworks.org)
+with details and reproduction steps. We will get back to you as soon as possible,
+and we may ask for additional information or guidance.
+
+Please avoid testing for security issues on the Archive of Our Own itself,
+as you risk disrupting other users and violating the [Terms of Service](https://archiveofourown.org/tos).
```

Comment:
Oh, should we link directly to the relevant section? https://archiveofourown.org/tos#IV.C.